media/js/base/protocol-globals.es6.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import MzpSupports from '@mozilla-protocol/core/protocol/js/supports';
+import MzpUtils from '@mozilla-protocol/core/protocol/js/utils';
+import MzpDetails from '@mozilla-protocol/core/protocol/js/details';
+
+window.MzpSupports = MzpSupports;
+window.MzpUtils = MzpUtils;
+window.MzpDetails = MzpDetails;
```

Comment:
**Curious: ** Why are these three the only ones being added to the window here? Noticed that others like the `MzpLangSwitcher` aren't

Reply:
These are all modules that are commonly used by many scripts throughout the site, so it makes sense to bundle them once in a shared script. If we imported them every time a file needed them, then we'd likely end up with larger bundles and duplicated code. Does that make sense?